conmon/conmon.c

```diff
@@ -152,11 +175,11 @@ int main(int argc, char *argv[])
 	/* Create the container */
 	if (terminal) {
 		snprintf(cmd, CMD_SIZE,
-			 "%s create %s --pid-file pidfile --console %s",
-			 runtime_path, cid, slname);
+			 "%s create %s %s --pid-file %s --console %s",
```

Comment:
> Actually I'm telling runC to run one container with a given bundle path and runC explicitly ignores the second argument.

I don't know where you got that impression from. That format has never been the case for runC (you've always specified a bundle with  `--bundle` or `-b`). Even the [currently unmerged CLI API RFC for runtime-spec](https://github.com/opencontainers/runtime-spec/pull/513/files#diff-0c1e5a5baa5649945697a530e09acbf5R41) uses the `--bundle` UX.

> AFAICT the runc create arg1 arg2 semantics are not defined anywhere in runc, and arg2 is neither interpreted as a container name nor as a bundle path. Am I missing something from the runC code ?

Ah, you're right. I was getting confused with `runc start`. Recently there's been a lot of PRs adding `runc <cmd> a b c d ...` support to a bunch of commands and I assumed that `runc create` was one of them.